### ISSUE TITLE
Documentation for @noRd functions 

### ISSUE THREAD

**Issue by CeresBarros** [bcgov/climR-pnw#110]
There are a few functions in `climr` that have little or no documentation and for which no documentation file (the .Rd file) is built (`@noRd`).

I think that it would be useful to try to 9almost) fully document these functions (description, parameters and imports), even if it's not going to be visible to package users, as this could facilitate future development (especially by new developers).

What do you think @kdaust, @cmahony ?

**Comment by kdaust** [comment]
Absolutely! But probably a lower priority than the ones that are accessible to users

**Comment by cmahony** [comment]
yes, agreed. thanks ceres.
